### PR TITLE
Update README.md with Hacktoberfest section

### PR DESCRIPTION
**Pull request by curquiza**
Update README according to hacktoberfest

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,14 @@ Meilisearch helps you shape a delightful search experience in a snap, offering f
 
 🔥 [**Try it!**](https://where2watch.meilisearch.com/) 🔥
 
+## 🎃 Hacktoberfest
+
+It’s Hacktoberfest 2022 @Meilisearch
+
+Hacktobearfest is a celebration of the open-source community. This year, and for the third time in a row, Meilisearch is participating in this fantastic event.
```
Comment (curquiza) [inline README.md:41]:
```suggestion
[Hacktobearfest](https://hacktoberfest.com/) is a celebration of the open-source community. This year, and for the third time in a row, Meilisearch is participating in this fantastic event.
```

File: README.md [hunk 2]
```diff
@@ -34,6 +34,14 @@ Meilisearch helps you shape a delightful search experience in a snap, offering f
 
 🔥 [**Try it!**](https://where2watch.meilisearch.com/) 🔥
 
+## 🎃 Hacktoberfest
+
+It’s Hacktoberfest 2022 @Meilisearch
+
+[Hacktobearfest](https://hacktoberfest.com/) is a celebration of the open-source community. This year, and for the third time in a row, Meilisearch is participating in this fantastic event.
```
Comment (brunoocasali) [inline README.md:41]:
```suggestion
[Hacktoberfest](https://hacktoberfest.com/) is a celebration of the open-source community. This year, and for the third time in a row, Meilisearch is participating in this fantastic event.
```

**Review by brunoocasali — APPROVED** [review]
Awesome! 🎉

**Review by Luna-meili — COMMENTED** [review]
Nothing more to add than Bruno's change, thanks so much for your help

**Comment by curquiza** [comment]
Thanks both of you, merging by hand again since Clippy is not happy!
not related with the code base

**Pull request merged**